docs/internationalization.adoc

```diff
@@ -1,28 +1,57 @@
 ## Instructions for internationalization
```

Comment:
Note: to review this file, please copy and paste it to a gist. The code blocks are not rendered correctly here in the review interface.